packages/slice-machine/src/icons/LogInIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import type { FC, SVGProps } from "react";
+
+export const LogInIcon: FC<SVGProps<SVGSVGElement>> = (props) => (
```

Comment:
We need to agree on the naming btw since remember I use @benjammartin naming. Anyway for me our naming need to be the same as Figma.

Reply:
Good call. 🙂 I changed it to "log in" since it is a verb, but "login" is also valid here and is what Material icons uses.

@xrutayisire Following Material is a good convention, in my opinion. I assume their team puts thought into the naming structure, or at least have their own conventions, which means we can offload our thinking to them.